docs/developer-docs/latest/development/plugins/i18n.md

```diff
@@ -0,0 +1,377 @@
+---
+title: Internationalization (i18n) - Strapi Developers Documentation
+description: Instructions on how to use Strapi Content API with the Internationalization (i18n) optional plugin
+---
+
+# Internationalization (i18n)
+<!-- ? wondering if it would be a good idea to add the 🌍 icon to the h1 — if yes, we should do this for the h1 of other plugins as well -->
```

Comment:
Yeah if go that route, reach plugin needs to have an emoji

Reply:
Do we agree that all doc pages will contain this suffix then: `- Strapi (Developers|User) Documentation`?
So another page with short title tag would be like: `GraphQL - Strapi Developers Documentation` in order to fix short title tags?

Reply:
I think the proper wording would be "Strapi Developer Documentation" and "Strapi User Guide" — I'd like @meganelacheny to confirm on her side :-)

I added this to prevent confusion, in case 2 entries share the same name in both user guide and dev docs. If you think this is unnecessary, please let me know ;-)

Reply:
LGTM